### PR TITLE
v97

### PR DESCRIPTION
**Pull request by Thorin-Oakenpants**
- 0605: see https://bugzilla.mozilla.org/show_bug.cgi?id=1355460

### CODE REVIEW

**Comment by szepeviktor** [comment]
> * 0605: see [bugzilla.mozilla.org/show_bug.cgi?id=1355460](https://bugzilla.mozilla.org/show_bug.cgi?id=1355460)

Mozilla guys are very tricky!

**Comment by Thorin-Oakenpants** [comment]
> Mozilla guys are very tricky!

I don't see an issue. About the only time I think this would trigger without you wanting to actually visit is when you left click to drag and drop. I only added it to be consistent

**Comment by Thorin-Oakenpants** [comment]
am done with #1366 and ready to merge - awaiting reviews

File: user.js [hunk 1]
```diff
@@ -452,11 +452,11 @@ user_pref("browser.shell.shortcutFavicons", false);
...
- * Blocks connections (SSL_ERROR_UNSAFE_NEGOTIATION) to servers that don't support RFC 5746 [2]
- * as they're potentially vulnerable to a MiTM attack [3]. A server without RFC 5746 can be
- * safe from the attack if it disables renegotiations but the problem is that the browser can't
- * know that. Setting this pref to true is the only way for the browser to ensure there will be
- * no unsafe renegotiations on the channel between the browser and the server.
+ * Blocks connections to servers that don't support RFC 5746 [2] as they're potentially vulnerable to a
+ * MiTM attack [3]. A server without RFC 5746 can be safe from the attack if it disables renegotiations
+ * but the problem is that the browser can't know that. Setting this pref to true is the only way for the
+ * browser to ensure there will be no unsafe renegotiations on the channel between the browser and the server
+ * [SETUP-WEB] SSL_ERROR_UNSAFE_NEGOTIATION: is it worth overriding this for that one site
```
Comment (fxbrit) [inline user.js:459]: add a question mark maybe?

**Review by fxbrit — APPROVED** [review]

**Comment by GlassGruber** [comment]
> > Mozilla guys are very tricky!
> 
> I don't see an issue. About the only time I think this would trigger without you wanting to actually visit is when you left click to drag and drop. I only added it to be consistent

Wouldn't this be triggered when reordering bookmarks?

**Comment by Thorin-Oakenpants** [comment]
> Wouldn't this be triggered when reordering bookmarks?

IDK and don't care, I disabled it :) I use a speed dial for my stuff, otherwise my toolbar has
- three folders and then four bookmarks - i never need to reorder anything - the rest of the space on the toolbar if for drg n drop URLs for me to follow up on etc
- i never use the bookmarks manager

So that's what I meant when I said drag n drop. I have no idea what you mean by "reordering bookmark" and how that differs from what I said

**Comment by GlassGruber** [comment]
> So that's what I meant when I said drag n drop. I have no idea what you mean by "reordering bookmark" and how that differs from what I said
> * i never use the bookmarks manager

I do sometimes use the bookmark manager and the usability... well is not really the main sell point of the tool IMO. It happens sometimes that I need to rearrange bookmarks that have been saved in weird order or similar. 

> IDK and don't care, I disabled it :) 
Disable first, ask questions later. Sounds like a good plan. 😎

**Comment by GlassGruber** [comment]
> Wouldn't this be triggered when reordering bookmarks?
> I have no idea what you mean by "reordering bookmark" and how that differs from what I said

Mmm I messed it a bit, I actually really intended a question but that negation morphed it in an assertion, correct phrase is:

Would this be triggered when reordering bookmarks?

Anyway yea, disabled and done.

**Review by earthlng — CHANGES_REQUESTED** [review]

**Review by earthlng — APPROVED** [review]

**Comment by crssi** [comment]
❤️

**Pull request merged**